hooks/tk-multi-publish2/basic/collector.py

```diff
@@ -159,7 +164,7 @@ def _collect_file(self, parent_item, path, frame_sequence=False):
             file_item.properties["is_sequence"] = True
             file_item.properties["sequence_files"] = [path]
 
-        self.logger.info("Collected file: %s" % (evaluated_path,))
+        self.logger.info("Collected file: {0} : {1}".format(evaluated_path,file_item))
```

Comment:
FYI, the `%` string formatting style is the current Toolkit standard. (I personally prefer `.format`).